CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Mapbox welcomes participation and contributions from everyone.
 - Added `guideMapUri` to the `RestStop`. [#6237](https://github.com/mapbox/mapbox-navigation-android/pull/6237)
 - [TileStore Android Service] Fixed a crash when the service process is killed by the Android system. [#6237](https://github.com/mapbox/mapbox-navigation-android/pull/6237)
 - Fixed `MapboxNavigationApp#detach` will not fully detach. This causes `MapboxNavigation` to continue to be accessible, and causes `MapboxNavigationObserver.onDetached` to be called multiple times.
+- Added Android 13 support. [#6196](https://github.com/mapbox/mapbox-navigation-android/pull/6196)
+- Declared [POST_NOTIFICATIONS](https://developer.android.com/reference/android/Manifest.permission#POST_NOTIFICATIONS) permission in SDK's AndroidManifest.xml. It is highly recommended for apps to request the permission in runtime, Without it the SDK will not be able to show the notification with trip progress in the notification drawer for apps that target Android 13 or higher. [#6196](https://github.com/mapbox/mapbox-navigation-android/pull/6196)
+- Bumped targetSdkVersion to 33. [#6196](https://github.com/mapbox/mapbox-navigation-android/pull/6196)
```

Comment:
Is this actually relevant to app developers? The app manifest has a higher priority and its `targetSdkVersion` will be chosen in the manifest merger either way.

Reply:
Removed it.